src/charm.py

```diff
@@ -93,6 +147,7 @@ def _on_update_status(self, event):
         - MySQL is not Initialized
         - Unit is active
         """
```

Comment:
Not sure if `start` is the right place for this:
From my observation, in the sequence of events on startup, `start` is **always** before `relation join/changed` and `pebble_ready` (correct me if I'm wrong).